support/fixtures/factories/facility.js

```diff
@@ -5,36 +5,64 @@ const faker = require('faker');
 const times = require('lodash.times');
 
 factory.define('facility')
+  .option('fromServer', false)
   .sequence('id')
   .attrs({
     address1: () => faker.address.streetAddress(),
     address2: () => faker.address.secondaryAddress(),
     city: () => faker.address.city(),
-    Info: () => {
+    createdAt: () => faker.date.past().toISOString(),
+    geometryId: () => faker.random.uuid(),
+    info: () => {
       return {
         primary_contact_email: faker.internet.email(),
         primary_contact_first_name: faker.name.firstName(),
         primary_contact_last_name: faker.name.lastName(),
         square_feet: faker.random.number()
       };
     },
-    Organization: () => factory.build('organization'),
     state: () => faker.address.state(),
     timezone: () => {
-      return faker.helpers.shuffle([
+      return faker.random.arrayElement([
         'America/New_York',
         'America/Chicago',
         'America/Denver',
         'America/Los_Angeles'
       ]);
     },
-    weather_location_id: () => null,
+    weatherLocationId: () => null,
     zip: () => faker.address.zipCode()
   })
   .attr('name', ['city'], (city) => `${faker.address.cityPrefix()} ${city}`)
-  .attr('organization_id', ['Organization'], (organization) => organization.id)
-  .attr('tags', ['id'], (id) => {
+  .attr('organization', ['fromServer'], (fromServer) => {
+    return factory.build('organization', null, { fromServer });
+  })
+  .attr('organizationId', ['organization'], (organization) => organization.id)
+  .attr('tags', ['id', 'fromServer'], (id, fromServer) => {
     return times(faker.random.number({ min: 0, max: 5 }), () => {
-      return factory.build('facilityTag', { facility_id: id });
+      return factory.build('facilityTag', { facilityId: id }, { fromServer });
     });
+  })
+  .after((facility, options) => {
+    // If building a facility object that comes from the server, transform it to have camel case and
+    // capital letters in the right spots
+    if (options.fromServer) {
+      facility.created_at = facility.createdAt;
```

Comment:
Doesn't `updated_at` also get returned?

Reply:
I'm not saying that it cannot be part of this data, but there isn't a single example of that has it on staging -- is there on production?

Reply:
Damn. Really!! One sec...

Reply:
Yeah, double-checked too. No `updated_at` gets returned; even though the data is stored in Postgres. Carry on!

Reply:
👍 We can definitely add it in the future if we need it